vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -32,6 +32,14 @@
 
 #define VIC_MPI_ROOT 0
 
+/******************************************************************************
+ *  * @brief   This structure stores netCDF file name and corresponding nc_id
+ *   *****************************************************************************/
+typedef struct {
+    char    nc_filename[MAXSTRING];
+    int     nc_id;
+} nameid_struct;
+
```

Comment:
The reason I put this here is that `vic_mpi.h` is lower level than `vic_driver_shared_image.h` (i.e., `vic_driver_shared_image.h` includes `vic_mpi.h`, but not vice versa); but the `get_scatter_nc_field_XX` functions in `vic_mpi.h` needs to use the nameid_struct structure. I agree that it's not super clean - do you have any suggestion?